packages/ui-components/src/lib/utils/handleDeployment.ts

```
@@ -0,0 +1,43 @@
+			});
+		} else {
+			handlers.handleDeployModal({
+				open: true,
+				args: {
+					...result,
+					subgraphUrl
+				}
+			});
+		}
+	} catch (e) {
+		throw e;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)